File: readability/readability.c
#include <cs50.h>
#include <stdio.h>

int count_letters(string text);

int count_words(string text);

int main(void)
{
   string text = get_string("Text: ");

   int letters = count_letters(text);

   int words = count_words(text);

   printf("%i letters\n",letters);

   printf("%i words \n", words);

}










int count_letters(string text)
{
    int letters=0;

    for (int i=0;text[i] != '\0'; i++)
    {
        if (((text[i] >= 65) && (text[i] <= 90)) || ((text[i] >=97) && (text[i] <=122)))
        {
            letters ++;
        }
    }
    return letters;
}


int count_words(string text)
{
    int words = 0;

    for (int i=0;text[i] != '\0'; i++)
    {
        if (text[i]==32)
        {
            words ++;
        }
    }
    return words;

}

File: sentimental-readability/readability.py
# TODO

from cs50 import get_string


storage =[]
input = get_string("Text: ")
for c in input:
    storage.append(c)

length = len(storage)
letters = 0
words = 0
sentences = 0

for counter in range(length):
    if storage[counter].isalpha():
        letters += 1
    elif storage[counter] == ' ':
        words += 1
    elif storage[counter] == '.' or storage[counter] == '!' or storage[counter] == '?':
        sentences += 1

words += 1

index = 0.0588 * ((letters/words) * 100) - 0.296 * ((sentences/words) * 100) - 15.8

